<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FGO Companion App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="name_txt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="star_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rarity</string>
    </property>
   </widget>
   <widget class="QLabel" name="class_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Class</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="asc1_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascension 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="star1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 *</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rarityButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="star2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2 *</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rarityButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="star3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3 *</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rarityButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="star4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4 *</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rarityButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="star5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5 *</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">rarityButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="saveServantButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Servant</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="id_num">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QComboBox" name="class_select">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="asc2_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascension 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="asc3_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascension 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="asc4_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascension 4</string>
    </property>
   </widget>
   <widget class="QComboBox" name="asc1_mat_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc1_qt_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc1_qt_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc2_qt_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc2_qt_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc2_qt_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc3_qt_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc3_qt_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc3_qt_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc4_qt_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc4_qt_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="asc4_qt_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QComboBox" name="asc1_mat_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc2_mat_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc3_mat_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sk_mats_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Skill Materials</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wiki_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiki Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wiki_txt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sk1_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1-&gt;2</string>
    </property>
   </widget>
   <widget class="QLabel" name="sk2_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2-&gt;3</string>
    </property>
   </widget>
   <widget class="QLabel" name="sk3_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3-&gt;4</string>
    </property>
   </widget>
   <widget class="QLabel" name="sk4_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4-&gt;5</string>
    </property>
   </widget>
   <widget class="QLabel" name="sk5_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5-&gt;6</string>
    </property>
   </widget>
   <widget class="QLabel" name="sk6_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6-&gt;7</string>
    </property>
   </widget>
   <widget class="QLabel" name="sk7_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7-&gt;8</string>
    </property>
   </widget>
   <widget class="QLabel" name="sk8_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>590</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8-&gt;9</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk1_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk1_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QComboBox" name="sk1_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk5_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk5_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk7_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk7_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk8_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>600</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk8_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>580</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QComboBox" name="asc4_mat_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc2_mat_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc2_mat_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc3_mat_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc3_mat_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc4_mat_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="asc4_mat_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk1_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk2_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk2_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk2_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk2_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QComboBox" name="sk3_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk3_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk3_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk4_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk4_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk6_qt_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sk6_qt_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>460</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sk3_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk4_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk4_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk5_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk5_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk6_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk6_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk7_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk7_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>540</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk8_mat_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sk8_mat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="costumeAddButton">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Costume</string>
    </property>
   </widget>
   <widget class="QListWidget" name="costumes_list">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="costumeEditButton">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>200</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Costume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="costumeDeleteButton">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>230</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Costume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewServantsButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>View Servants</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ascResetButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Ascension Materials</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sk_name_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sk_name_1_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First Skill:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sk_name_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sk_name_2_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Second Skill:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sk_name_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sk_name_3_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Third Skill:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="skResetButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Skill Materials</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveExitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save and Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rarityButtons"/>
 </buttongroups>
</ui>
